generative_bayes_model/gbm-run.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+if [[ -z "${S3_BUCKET-}" ]]; then
+    echo "No S3 bucket provided (use env var S3_BUCKET)"
+    exit 1
+fi
+
+echo "Running pymc3_generative_model..."
+python3 pymc3_generative_model.py;
+
+echo "Copying results to S3 bucket..."
+if [[ -n "$S3_BUCKET" ]]; then
+    aws s3 sync "." "$S3_BUCKET/generative_bayes_model/" "--exclude" "*" "--include" "*.svg"
```

Comment:
do you need all those `"`?

Reply:
what's `--exclude *` for?

Reply:
i think `aws s3 cp *.svg $S3_BUCKET/generative_bayes_model/` might work 🤔

Reply:
I tried the same (but with quotes) and it didn't work! I thought quotes were a part of the syntax. So I did some reading and was told that, you will choose the src and destination paths, then you'll exclude everything using a wildcard and include only those files that you want to copy/sync. I can try a version of it without the quotes as well! Do you want me to?

Reply:
i think we can use `cp` instead of `sync` for this use case

Reply:
Pasting aws cli docs for reference -- https://docs.aws.amazon.com/cli/latest/reference/s3/#use-of-exclude-and-include-filters